test/static/turtle_article_case.html

```diff
@@ -13,6 +13,8 @@
 <meta name="Description" content="Exposition on the awesomeness of turtles"/>
 <meta name="Keywords" content="turtles, are, awesome" />
 
+<link rel="APPle-touch-icon" href="turtleapple.png" sizes="72x72">
+<link rel="IcOn" href="turtle.png" sizes="18x18" type="image/png">
```

Comment:
It might be a good idea in this or the other to test what happens if sizes = "", this is one of the errors that the tests didn't catch.